src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,57 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+function grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+    return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+end
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal([ring::MPolyRing], subspace_dimension::Int, ambient_dimension::Int)
```

Comment:
Elsewhere we put the `,` into the optional part, i.e.,
```suggestion
    grassman_pluecker_ideal([ring::MPolyRing,] subspace_dimension::Int, ambient_dimension::Int)
```
(And the same in the .md file then)